cves/CVE-2017-9996.yml

```diff
@@ -64,7 +64,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  .cdxl files are a type of video format that was used to display video on the
+  Commodore Amiga. This vulnerability is caused by the .cdxl decoding not
+  checking if the file being decoded was in the CHUNKY format. Not checking
```

Comment:
Might want to **briefly** explain the CHUNKY format if you can find information on it, and how/what you need to change to cause this vulnerability to be an issue; ie what exactly causes the buffer overflow error.